pkg/ffapi/restfilter_json.go

```diff
@@ -210,7 +210,13 @@ func (jq *QueryJSON) addSimpleFilters(ctx context.Context, fb FilterBuilder, jso
 func joinShortNames(long, short, negated []*FilterJSONKeyValue) []*FilterJSONKeyValue {
 	res := make([]*FilterJSONKeyValue, len(long)+len(short)+len(negated))
 	copy(res, long)
-	copy(res[len(short):], short)
```

Comment:
```suggestion
	copy(res[len(long):], short)
```

Isn't this the bug @SamMayWork ?

Reply:
(if there were a problem with `copy` having `nil` as the right parameter, then that would be a problem in the line above, which this PR doesn't change)

Reply:
Yep after making the change from `short -> long` the surrounding if block is actually redundant now, removing now 🖊️

Reply:
Resolved in https://github.com/hyperledger/firefly-common/pull/124/commits/066994cbb94c663b8f892f44f9c1bd248db13db9